### ISSUE TITLE
the code can not be recognizerd 

### ISSUE THREAD

**Issue by santcool** [cssivision/react-native-qrcode#19]
I use it like this:            <QRCode value={'http://www.baidu.com/'}
                    bgColor="#fff"
                    fgColor="#333"/></View>)

**Comment by mltucker** [comment]
I think you need to switch your bgColor and fgColor.

**Comment by cssivision** [comment]
in some Extreme situation, this may happen, but i can not fix this.

**Comment by Raven11** [comment]
Facing similar issue. QrCode generated is not recognized.

**Comment by froatsnook** [comment]
@Raven11 what are your `fgColor` and `bgColor`?

**Comment by Raven11** [comment]
<QRCode
          value={this.props.dataSource}
          size={200}
          bgColor='black'
          fgColor='white'/>

**Comment by froatsnook** [comment]
@Raven11 I think that's just too much data.  I was able to get it to scan when I used `size=300` and `value='{"orderIdPOS":"41257","orderIdFL":1057,"customerId":119746142258,"fulfillType":0,"storeCode":"0111001228","grossAmount":"187","netAmount":"155.00","totalTax":"32.00","totalDiscount":"0","cash":"173.0","shots":"14.0"}'` (i.e. I cut out items and the base64 at the end -- also unescaped the `"` since you used a `'` quote, but that shouldn't really matter).
